client/components/seo/facebook-preview/index.jsx

```diff
@@ -32,6 +32,8 @@ export class FacebookPreview extends PureComponent {
 	render() {
 		const { url, type, title, description, image, author } = this.props;
 
+		const strippedDescription = description ? description.replace( /<[^>]+>/g, '' ) : '';
```

Comment:
I did a little research about striping HTML, and found this function in [a CSS tricks post](https://css-tricks.com/snippets/javascript/strip-html-tags-in-javascript/):

```es6
const strippedString = originalString.replace(/(<([^>]+)>)/gi, "");
```

Just FYI. Also, wondering if we could create a single helper function for this,

Reply:
> Also, wondering if we could create a single helper function for this.

I will create a follow up PR cause I just saw it when hitting the merge button!